pkg/deploy/generator/scripts/util-services.sh

```diff
@@ -25,27 +25,33 @@ enable_services() {
 # 1) image - nameref, string; container image
 # 2) role - nameref, string; VMSS role
 # 3) conf_file - nameref, string; aro gateway environment file
+# 4) ipaddress - nameref, string; static ip of podman network to be attached
 configure_service_aro_gateway() {
     local -n image="$1"
     local -n role="$2"
     local -n conf_file="$3"
+    local -n ipaddress="$4"
     log "starting"
     log "Configuring aro-gateway service"
 
     local -r aro_gateway_conf_filename='/etc/sysconfig/aro-gateway'
+    local -r add_conf_file="PODMAN_NETWORK='podman'
+IPADDRESS='$ipaddress'
+ROLE='${role,,}'"
```

Comment:
What is the purpose of this?

Uppercase variables are meant to communicate that a variable is used externally. `role` is used internal to this function.

Reply:
We could get rid of the initial assignment and directly reference the positional arguments like 

```
IPADDRESS='$4'
ROLE='$2'"
```

but that would not have much of an improvement over readability of the code.

Reply:
> but that would not have much of an improvement over readability of the code.  
```bash
    local -n image="$1"
    local -n role="$2"
    local -n conf_file="$3"
    local -n ipaddress="$4"
```
Right, the numbered variables are already assigned to named varaibles, I think it's overly complicated to re-name them again with a new set of variables, when `ipaddress` and `role` are never directly used.

My recommendation is to use the first assignment variables, there's no need to copy them again.

Reply:
Thanks for explaining. My bad, I read this quickly and didn't realize they are being assigned to those values in the `EnvironmentFile`, not in the current shell context.